qgreenland/config/layers.yml

```diff
@@ -1,8 +1,3 @@
-# TODO: Consider removing short_name key and just use the layer key
-# TODO: Consider renaming file_type to "output_file_type" or something --
-# currently this is used when adding layers to the project file, so we should
-# consider this as output_file_type and use a new key for source file type.
-
```

Comment:
Extracted to the schema, because these are schema changes

Reply:
I accidentally made this as a review instead of a single comment because that's apparently what happens when you hit CTRL+Enter after writing a comment! Not what I wanted :(